### ISSUE TITLE
Invisible (Colored) Text and UI Elements in Windows 11 22H2 (Build 22621.521)

### ISSUE THREAD

**Issue by titan58002** [MicaForEveryone/MicaForEveryone#186]
The Issue is that Enabling MicaForEveryone causes the Text and UI elements in "Windows Explorer" and most third party apps to change from white to colored and therefore invisible. I wanted to also include a screenshot of the issue but in trying so I found out that the issue is not seen in screenshots of the window! (the invisibility problem doesn't exist in screenshots of the different windows (both win explorer and third party apps) and everything is perfect! weird!)
I tried to use another tool that offers similar service (https://github.com/Maplespe/ExplorerBlurMica) (its only for windows explorer and not third party apps) and the issue is partly solved when using this tool alongside MicaForEveryone and after enabling this tool mica starts to work fine in windows explorer but the issue remains in third party app windows. (I know this maybe unrelated to MicaForEveryone itself but I added for more context)

**Comment by xmine64** [comment]
Read the readme, search in the issues, check FAQ.

TLDR: not possible with MFE.
